x-pack/plugins/apm/public/components/shared/Links/apm/ServiceMapLink.tsx

```diff
@@ -19,6 +20,6 @@ interface ServiceMapLinkProps extends APMLinkExtendProps {
 }
 
 export function ServiceMapLink({ serviceName, ...rest }: ServiceMapLinkProps) {
-  const path = pathFor(serviceName);
```

Comment:
You should inline `pathFor` in `useServiceMapHref` since it's not re-used and doesn't provide any value on its own.